### DISCUSSION TITLE
NPS error: The revocation function was unable to check revocation for the certificate.

### DISCUSSION THREAD

**Discussion by Tuteksik** [chris2511/xca#431] [Q&A]
Hi,
I would to use xca ca certificate on the windows NPS server, I have configured CRL and put this CRL file on the IIS server, the file is accessible:
`Verify cert:  certutil.exe -verify -urlfetch .\Root_CA.crt
----------------  Certificate CDP  ----------------
  Verified "Base CRL (01)" Time: 0 a95489d7bf1887e7cd8f2a4ff5257cb1536bd312
    [0.0] http://serv18.ad.domain.pl/crl/ca.crl`

But NPS during connection give me Error 6273 reason 258: The revocation function was unable to check revocation for the certificate.

**Comment by chris2511** [comment]
I don't know anything about this topic. I even had to google `Windows NPS server`
Maybe somebody else can help.
